app/src/shared/ui/HeaderChat/HeaderChat.tsx

```diff
@@ -0,0 +1,55 @@
+import { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import MoreOutlined from '@ant-design/icons/MoreOutlined';
+
+import { Avatar } from 'ui-kit';
+import { ActionItem } from 'ui-kit/Actions';
+import { AvatarShapes } from 'ui-kit/Avatar';
+import { DropdownMenu } from 'ui-kit/DropdownMenu';
+import { IconButton, IconButtonSize } from 'ui-kit/IconButton';
+import { PopoverPosition, PopoverSide } from 'ui-kit/Popover';
+
+import css from './HeaderChat.module.scss';
+
+interface HeaderChatProps {
+    label: string;
+    actions: ActionItem[];
+    avatarShape?: AvatarShapes;
+    avatarSrc?: string;
+}
+
+const HeaderChat: FC<HeaderChatProps> = (props) => {
+    const { t } = useTranslation();
+    const dropdownActions = props.actions.map((item) => ({
```

Comment:
```suggestion
    const { t } = useTranslation();
    
    const dropdownActions = props.actions.map((item) => ({
```
энтеры можно использовать для группировки

Reply:
почему бы не мемо